### ISSUE TITLE
Different line breaks for synced and unsynced lyrics?

### ISSUE THREAD

**Issue by coolpringles** [tranxuanthang/lrcget#117]
Let's say I'm transcribing a song that has the chorus start very close after a verse. When syncing the lyrics, I'd time it without a space between the lines, but for unsynced lyrics I'd want to keep a line break there. Is there a way to do this in lrcget?

**Comment by alexlambson** [comment]
Yeah, make the next line have the same timestamp as the blank line.

```lrc
[00:32.00] No chance you can make this right
[00:35.88]
[00:35.88] I know you wanna keep me buried
```

In a music player it will stay on `No chance you can make this right`, then skip the blank line and jump to `I know you wanna keep me buried`.
